model/textparse/openmetricsparse.go

```diff
@@ -222,9 +225,108 @@ func (p *OpenMetricsParser) Exemplar(e *exemplar.Exemplar) bool {
 // CreatedTimestamp returns nil as it's not implemented yet.
 // TODO(bwplotka): https://github.com/prometheus/prometheus/issues/12980
 func (p *OpenMetricsParser) CreatedTimestamp() *int64 {
+	var lbs labels.Labels
+	p.Metric(&lbs)
+	lbs = lbs.DropMetricName()
+
+	newParser := deepCopyParser(p)
+loop:
+	for {
+		switch t, _ := newParser.Next(); t {
+		case EntrySeries:
+			// TODO: potentially broken? Missing type?
```

Comment:
Is this comment still relevant?

Reply:
Hmm, might still be relevant
We're not sure if TYPE text is optional yet
if it is how do we handle it